File: js/problem2.js
var sum = 0;
var num1 = 1, num2 = 2, temp = 1;

while(num2 <= 4000000)
{
	if(num2 % 2 === 0)
	{
		sum += num2;
	}
	temp = num1;
	num1 = num2;
	num2 += temp;
}

document.write("<h2>Solution to Problem 2: "+sum+"</h2>");

File: java/problem2.java
class Problem2{
	public static void main(String args[]){
		int sum=0; 
		int num1=1; 
		int num2=2; 
		int temp=1;

		while(num2 <= 4000000){
			if(num2%2==0){
				sum+=num2;
			}
			temp = num1;
			num1 = num2;
			num2 += temp;
		}
		System.out.println(sum);
	}
}
